UI/MultiplayerLobbyWnd.cpp

```diff
@@ -676,10 +695,10 @@ void MultiPlayerLobbyWnd::DoLayout() {
     GG::Pt players_lb_lr = players_lb_ul + GG::Pt(ClientWidth() - CONTROL_MARGIN - x, m_chat_input_edit->RelativeUpperLeft().y - CONTROL_MARGIN - y);
     m_players_lb->SizeMove(players_lb_ul, players_lb_lr);
 
-    m_start_game_bn->SizeMove(GG::Pt(GG::X0, GG::Y0), GG::Pt(GG::X(125), m_start_game_bn->MinUsableSize().y));
-    m_cancel_bn->SizeMove(GG::Pt(GG::X0, GG::Y0), GG::Pt(GG::X(125), m_start_game_bn->MinUsableSize().y));
+    m_ready_bn->SizeMove(GG::Pt(GG::X0, GG::Y0), GG::Pt(GG::X(125), m_ready_bn->MinUsableSize().y));
+    m_cancel_bn->SizeMove(GG::Pt(GG::X0, GG::Y0), GG::Pt(GG::X(125), m_ready_bn->MinUsableSize().y));
     m_cancel_bn->MoveTo(GG::Pt(ClientWidth() - m_cancel_bn->Width() - CONTROL_MARGIN, ClientHeight() - m_cancel_bn->Height() - CONTROL_MARGIN));
-    m_start_game_bn->MoveTo(GG::Pt(m_cancel_bn->RelativeUpperLeft().x - CONTROL_MARGIN - m_start_game_bn->Width(),
+    m_ready_bn->MoveTo(GG::Pt(m_cancel_bn->RelativeUpperLeft().x - CONTROL_MARGIN - m_ready_bn->Width(),
                                    ClientHeight() - m_cancel_bn->Height() - CONTROL_MARGIN));
```

Comment:
Align with start of parameter list (or indented one level for long lines)
`ClientHeight` should align with `m_cancel_bn` in this case